R/bundle.R

```diff
@@ -8,6 +8,13 @@ bundleAppDir <- function(appDir, appFiles) {
   for (file in appFiles) {
     from <- file.path(appDir, file)
     to <- file.path(bundleDir, file)
+    # if deploying a single-file Shiny application, name it "app.R" so it can
+    # be run as an ordinary Shiny application
+    if (!is.null(appPrimaryDoc) &&
```

Comment:
If we're checking the `NULL`-ness of `appPrimaryDoc` here should we set the default value as `NULL`? (Also defends us against any other calls for `bundleAppDir` that don't currently specify it)